app/models/location.rb

```diff
@@ -1,16 +1,20 @@
 class Location < ApplicationRecord
 
+  belongs_to :county, optional: true
```

Comment:
reason is it's bugged right now and not fully working, and is forcing a connected county while i can't generate one in seeds.rb for some reason so i made it optional to populate the database. we should change that later